src/app/common/stores/applications/SsvWeb/OperatorMetadata.store.ts

```diff
@@ -142,11 +143,13 @@ class OperatorMetadataStore extends BaseStore  {
                 } else {
                     this.setErrorMessage(metadataFieldName, '');
                 }
-            } else if ([FIELD_KEYS.LINKEDIN_URL, FIELD_KEYS.WEBSITE_URL, FIELD_KEYS.TWITTER_URL].includes(metadataFieldName) && typeof fieldEntity.value === 'string') {
-                const res = isLink(fieldEntity.value);
+            } else if ([FIELD_KEYS.LINKEDIN_URL, FIELD_KEYS.WEBSITE_URL, FIELD_KEYS.TWITTER_URL, FIELD_KEYS.DKG_ADDRESS].includes(metadataFieldName) && typeof fieldEntity.value === 'string') {
+                const isDkgField = metadataFieldName === FIELD_KEYS.DKG_ADDRESS;
+                const res = isDkgField ? checkDkgAddress(fieldEntity.value) : isLink(fieldEntity.value);
+                const errorMessage = isDkgField ? translations.OPERATOR_METADATA.DKG_ADDRESS_ERROR : translations.OPERATOR_METADATA.LINK_ERROR;
```

Comment:
This can be refactored a bit.
Instead of having such a big condition and then isDkgField ? ... : ... 
There should be some structured code when you can just pass metadataFieldName and fieldEntity to that structure/method, and you will have final res and errorMessage as returned value.

Reply:
refactored